src/backend/vulkan/src/lib.rs

```diff
@@ -511,6 +511,11 @@ impl hal::PhysicalDevice<Backend> for PhysicalDevice {
     }
 
     fn features(&self) -> Features {
+        // see https://github.com/gfx-rs/gfx/issues/1930
+        let is_windows_intel_kaby = cfg!(windows) &&
+            self.properties.vendor_id == 0x8086 &&
+            self.properties.device_id >> 2 == 0x59;
```

Comment:
How does is the Intel device ID built up? 2-bit shift compared with a hex value looks kind of suspicious.